simulation/settings.py

```diff
@@ -0,0 +1,24 @@
+import argparse
+import datetime
+
+import pandas as pd
+
+
+def parse_args(args=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--household-distribution", type=pd.read_csv)
+    return parser.parse_args(args)
+
+
+args = parse_args()
```

Comment:
If you have this here then as soon as anything is imported from this module, `parse_args` will be excuted and try to parse arguments. Instead `__main__` should import `get_args` and call it there.

I know a settings file has been mentioned a lot lately. I'm not convinced it's a panacea.

Genuine constants should be constants in the appropriate Python module.

Things like `NUM_STEPS`, `START_DATETIME` and `NUM_HOUSEHOLDS` are simulation parametes, so I think they should go in `parse_args` with sensible defaults, so when the user runs the simulation they can be changed on the command line.

If everyone really wants to specify things in a file, then we could have an argument for a file path and load settings from that instead of the command line.